mariadb-image-builder

```diff
@@ -162,6 +162,8 @@ BUILDER_DOCKER_HOST=$(projectEnvironmentVariableCheck BUILDER_DOCKER_HOST "${BUI
 
 BUILDER_MTK_YAML_BASE64=$(projectEnvironmentVariableCheck BUILDER_MTK_YAML_BASE64 "${BUILDER_MTK_YAML_BASE64}")
 
+# optional
+BUILDER_ONLY_LATEST=$(projectEnvironmentVariableCheck BUILDER_ONLY_LATEST "0")
```

Comment:
I think this could be changed to be multiple options, because it may be possible that someone wants to push one or the other, or both.
```
BUILDER_PUSH_TAGS=$(projectEnvironmentVariableCheck BUILDER_PUSH_TAGS "both")
```

Then we can allow for the options of
* latest - push only the latest tag
* default - push only the tag that would be created by the `BUILDER_BACKUP_IMAGE_TAG`
* both

and then the push logic would be
```
if [ "$BUILDER_PUSH_TAGS" == "both" ] || [ "$BUILDER_PUSH_TAGS" == "latest" ]; then
	docker push "${BUILDER_BACKUP_IMAGE_NAME}:latest"
fi
if [ "$BUILDER_PUSH_TAGS" == "both" ] || [ "$BUILDER_PUSH_TAGS" == "default" ]; then
	docker push "${backup_image_full}"
fi

Reply:
Cool beans — yeah I made the assumption that we would always want to push `latest`. I'll update so that is more flexible.

Reply:
Awesome! Thanks